docs/source/scripts/plot_contour.py

```diff
@@ -0,0 +1,20 @@
+import optuna
+import plotly
+
+
+def main():
+    def objective(trial):
+        x = trial.suggest_uniform('x', -100, 100)
+        y = trial.suggest_categorical('y', [-1, 0, 1])
+        return x ** 2 + y
+
+    study = optuna.create_study()
+    study.optimize(objective, n_trials=10)
```

Comment:
```suggestion
    sampler = TPESampler(seed=10)
    study = optuna.create_study(sampler=sampler)
    study.optimize(objective, n_trials=10)
```
@hvy Does this change make the result deterministic?

Reply:
Yes, that should be enough (besides the fact that you need to import the `TPESampler`).